std_msgs/Header header
float64[3] error_pos_EE
float64[3] dot_error_pos_EE
float64[7] dot_qr
float64[7] ddot_qr
float64[7] s

float64[10] Link1_param
float64[10] Link2_param
float64[10] Link3_param
float64[10] Link4_param
float64[10] Link5_param
float64[10] Link6_param
float64[10] Link7_param

float64[7] tau_cmd
